target/linux/ramips/dts/rt5350_olimex_rt5350f-olinuxino-evb-plus.dts

```diff
@@ -0,0 +1,119 @@
+/dts-v1/;
+
+#include "rt5350_olimex_rt5350f-olinuxino-evb.dts"
```

Comment:
Please do not include another DTS, but create a DTSI for the shared code that is included in DTSes for both devices then.

Reply:
Just had a closer look and realized that a rt5350_olimex_rt5350f-olinuxino.dtsi is already there. So, base the new device on this one. If you still would need to delete &spi0 there, just move it from DTSI to rt5350_olimex_rt5350f-olinuxino.dts and rt5350_olimex_rt5350f-olinuxino-evb.dts instead.